os/arch/arm/src/armv7-a/arm_assert.c

```diff
@@ -636,6 +625,13 @@ void up_assert(const uint8_t *filename, int lineno)
 	lldbg_noarg("\n");
 #endif
 
+#ifdef CONFIG_SMP
+	/* If SMP is enabled then we need to resume all the other cpu's
+	 * which we paused earlier.
+	 */
+	up_cpu_resume_all();
+#endif
```

Comment:
Is this right to resume all cpus before recovering the fault?

Reply:
So, there will be two cases:
1) If it is kernel fault, then the board will reboot. Resuming all CPUs won't affect this case.
2) If the crash is in app binary, then we will be pausing all the CPUs inside recovery.

Reply:
1. Even we call reboot, why do we need to resume?
2. Who(where) pauses the all cpus in recovery?

Reply:
1. Yeah, I agree. There is no point in resuming here for this case.
2.  As, we are not pausing the CPUs in recovery as of now, let me revert this change.

Reply:
2. When app fault is done in recovery, all the cpus should be resumed. You should add resuming at somewhere.

Reply:
Let me check it properly.